sklearnex/tests/test_memory_usage.py

```diff
@@ -124,6 +132,32 @@ def gen_functions(functions):
 ORDER_DICT = {"F": np.asfortranarray, "C": np.ascontiguousarray}
 
 
+DUMMY_ESTIMATOR = {}
+if _is_dpc_backend:
+
+    from onedal.datatypes._data_conversion import from_table, to_table
```

Comment:
I don't think an underscore in the filename is the right way, if you want to have an underscore, then I would make an ```__init__.py``` in the datatypes folder, and expose from_table and to_table there.